core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/OracleExportAction.java

```diff
@@ -137,17 +138,15 @@ private String buildOracleExportCommand() {
   }
 
   private String getColSeparator(String format) {
-    switch (SeparatorFormat.valueOf(format)) {
+    switch (SeparatorFormat.valueOf(format.toUpperCase())) {
       case CSV:
         return ",";
       case TSV:
         return "  ";
       case PSV:
         return "|";
-      default:
-        throw new IllegalArgumentException(
-          String.format("Invalid format '%s'. Must be one of %s", format, EnumSet.allOf(SeparatorFormat.class)));
     }
+    return null;
```

Comment:
my bad. you needed the default case because you had to return a ``String`` from this method. Can you please revert this change to bring back the default case?